### DISCUSSION TITLE
Wrong generation link for last build  in "Build status badge"

### DISCUSSION THREAD

**Discussion by hodev** [codemagic-ci-cd/codemagic-docs#2601] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Other - please mention in your question

### Steps to reproduce

1. Build project
2. Open Workflow settings
3. Copy code from "Badge markdown"
4. Insert code in any readme.md
5. Try to open link from badge

### Expected results

1. Last build should be opened. Link should be like as https://codemagic.io/app/xxxxxxx/latest_build instead of 
 https://codemagic.io/apps/xxxxxxx/latest_build (wrong **s** in app**s**)

### Actual results

1. Opened page with error "Not found
The page you are looking for does not exist."

**Comment by dam-ease** [comment]
Thanks for the report @hodev!
This has been reproduced, and the team is looking into it.

**Comment by dam-ease** [comment, marked as answer]
Hello @hodev.
This has been fixed now!
<details><summary>Screenshot</summary>
<p>

![Screenshot 2024-06-20 at 15 51 25](https://github.com/codemagic-ci-cd/codemagic-docs/assets/53122008/e02052f8-e9bb-48d3-9ad2-3917d5e305e3)

</p>
</details> 

I'll be closing this from here. Thank you!
